python/mxnet/gluon/block.py

```diff
@@ -967,6 +968,30 @@ def _build_cache(self, *args):
                 self._cached_op_args.append((False, params[name]))
         flags = [('data_indices', data_indices), ('param_indices', param_indices)] + \
                 self._flags
+
+        args, _ = _flatten(args, "input")
+        args_without_none = [ele for ele in args if ele is not None]
+        try:
+            cargs = [args_without_none[i] if is_arg else i.data()
```

Comment:
cargs is a local variable and is not used in this function. Does it still make sense to keep cargs here?